apps/bfd-db-migrator/src/main/assembly/bin/bfd-db-migrator.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 
+set -x
```

Comment:
Likely a debugging line left behind...
```suggestion
```

Reply:
We'll need to change it before long... the somewhat naive .json log that we're producing gets populated with this verbose output. Nothing's consuming that json output yet so I think it's harmless for the here and now.